content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -413,32 +420,31 @@ Istio.
 
 In this task you looked at three ways to call external services from an Istio mesh:
 
-1. Using a `ServiceEntry` for HTTP and a combination of `ServiceEntry` and `VirtualService` for HTTPS. This is the
-   recommended way.
+1. Using a `ServiceEntry` to register an accessible external service inside the mesh. This is the
```

Comment:
Don't use `ServiceEntry` to refer to all service entries. It is best to use it when referring to a specific instance in a yaml file. Opt instead for the normal service entry. 

```suggestion
1. Use a service entry to register an accessible external service inside the mesh. This is the
```